src/cli/Cli.ts

```diff
@@ -947,6 +947,22 @@ export class Cli {
     return response;
   }
 
+  public static async interactivePrompt(interactive: boolean, promptMessage: string, errorMessage: string, values: any[]): Promise<string> {
```

Comment:
`values` shouldn't be `any` here. As we're passing them into inquirer, they should match the format of the options expected by inquirer. Better yet, we should pass them as a key-value pair, where the key is the value that we use for disambiguation and value is what we show to the user. They can be the same but don't have to be.